composer.lock

```diff
@@ -0,0 +1,2803 @@
+{
```

Comment:
lock file should not commited currently, but also not in ignore list. so please remove this file for now as it contains wrong depencies